src/server/get_head_html.js

```diff
@@ -0,0 +1,12 @@
+import path from 'path';
```

Comment:
Shall we move this into a file called `src/server/utils.js` and export the following function as `getHeadHtml`.